patches/minecraft/net/minecraft/client/multiplayer/PlayerController.java.patch

```diff
@@ -96,12 +96,11 @@
        }
  
        return p_178893_1_.equals(this.field_178895_c) && flag;
-@@ -257,13 +264,27 @@
+@@ -257,13 +264,26 @@
           return ActionResultType.FAIL;
        } else {
           ItemStack itemstack = p_217292_1_.func_184586_b(p_217292_3_);
-+         net.minecraftforge.event.entity.player.PlayerInteractEvent.RightClickBlock event = net.minecraftforge.common.ForgeHooks
-+                 .onRightClickBlock(p_217292_1_, p_217292_3_, blockpos, p_217292_4_.func_216354_b());
++         net.minecraftforge.event.entity.player.PlayerInteractEvent.RightClickBlock event = net.minecraftforge.common.ForgeHooks.onRightClickBlock(p_217292_1_, p_217292_3_, blockpos, p_217292_4_.func_216354_b());
```

Comment:
This is fine, but it also is wrapped to make reading easier. Doesn't really matter tho.
This event needs a re-work...